namespaces/live-1.cloud-platform.service.justice.gov.uk/track-a-move-dev/resources/acm.tf

```diff
@@ -0,0 +1,18 @@
+resource "aws_acm_certificate" "apigw_custom_hostname" {
```

Comment:
Nobody is monitoring ACM certificate renewals, when this expires it will do so silently (to my knowledge). This there anyway to use the already provided and maintained cert-manager instance or not have a custom hostname?

Reply:
In fact, this might be why you put the below validation? Can you confirm @mafonso